### ISSUE TITLE
Paginate from oldest to newest when doing catch up queries

### ISSUE THREAD

**Issue by poplexity** [telosnetwork/obe-indexer#18]
Not sure if there's a limit param we could set, but this might not get all the deltas... and if it didn't get them all, it wouldn't paginate well.  It should sort oldest to newest and work it's way forward, or visa versa.

https://github.com/telosnetwork/obe-indexer/blob/b0735debe1b664dc7419d531090c668f0271d6df/src/indexer/jobs/token/TokenPoller.ts#L278

**Comment by tomtomcrypto** [comment]
```
        const params = {
            after: startISO,
            before: endISO,
            code: token.account,
            table: 'accounts',
            sort: 'asc',
            limit: 1000
        };
```

That is what I use for actions, should work for deltas I guess. We just need to make sure there isn't > 1000 actions in the pooling interval or it will take some time to catch up. Or increase it now, I'll make it configurable given it needs to be implemented on the deltas for tokens too.

**Comment by tomtomcrypto** [comment]
Yeah it is badly documented (not at all) but it should work https://github.com/eosrio/hyperion-history-api/blob/91fd57bd8373396d7ae55d1a7eef41980d2d2411/api/routes/v2-history/get_deltas/get_deltas.ts#L28
